docs/source/_developers/primer.md

```diff
@@ -0,0 +1,234 @@
+# Documentation Primer
+
+:::{important}
+Remember that this style guide is a reference for team members working on documentation. It is essential to keep it up-to-date and easily accessible for all contributors.
+:::
+
+This style guide outlines the best practices and conventions for creating clear, consistent, high-quality documentation for our project. By following these guidelines, we can ensure a cohesive writing style and tone across all documentation.
+
+This style guide provides guidelines for writing clear, concise, and consistent documentation. It covers language, tone, formatting, and other aspects of documentation.
+
+##  Writing Style
+The writing style in documentation is critical for ensuring consistency and understanding. Here are some best practices:
+
+**Clarity**: Be as clear and straightforward as possible. Avoid ambiguity. Documentation is not a place for complex sentences and unusual words.
+
+:::{list-table}
+:header-rows: 1
+:widths: 50 50
+:align: left
+
+* - ❌ Do Not
+  - ✅ Do
+* - The initiation of the replication process...
+  - To start the replication process...
+:::
+
+**Conciseness**: Keep sentences and paragraphs short. This makes text easier to read and understand.
+
+:::{list-table}
+:header-rows: 1
+:widths: 50 50
+:align: left
+
+* - ❌ Do Not
+  - ✅ Do
+* - Once the process has been completed successfully, the next step is...
+  - After completing the process, next...
+:::
+
+**Active Voice**: Use the active voice whenever possible as it is more straightforward and engaging.
+
+:::{list-table}
+:header-rows: 1
+:widths: 50 50
+:align: left
+
+* - ❌ Do Not
+  - ✅ Do
+* - The command can be executed by the user.
+  - The user can execute the command.
+:::
+
+**Address the Reader**: Use the second person ("you") to address the reader directly.
+
+:::{list-table}
+:header-rows: 1
+:widths: 50 50
+:align: left
+
+* - ❌ Do Not
+  - ✅ Do
+* - One can see the results by... <br>
+  - You can see the results by...<br>
+:::
+
+- Avoid using jargon and slang, as they can be confusing.
+- Keep your writing easy to read by using short sentences and paragraphs.
```

Comment:
This is stated about in the Do and Do Not table so we can probably strike it here.